src/app/manager/manager.component.scss

```diff
@@ -0,0 +1,6 @@
+@import '../variables.scss';
```

Comment:
Same as the other `.scss` file, don't copy just reference the file that you need the styles from.